### DISCUSSION TITLE
Self-Bots

### DISCUSSION THREAD

**Discussion by jateute** [Hyperclaw79/PokeBall-SelfBot#208] [Q&A] [answered]
I think self-bots aren't allowed on Discord and and will be reasoned with an account blocking?

**Comment by Hyperclaw79** [comment, marked as answer]
Well, it's against the ToS and I have already explicitly mentioned it in the Readme. But then again, I rarely heard of these kinda selfbots getting banned till now. The ones which get banned are raidbots and such.

**Reply by jateute** [reply]
Sorry, I didn't see that.
